src/markdown.cpp

```diff
@@ -551,6 +551,10 @@ size_t Markdown::Private::isSpecialCommand(std::string_view data,size_t offset)
       // skip over name (and optionally type)
       while (offset_<data_.size() && (c=data_[offset_])!='\n' && (allowSpaces || c!=' ') && c!='(')
       {
+        if (offset_<data_.size()-9 && data_[offset_]=='\\')
```

Comment:
Please use `offset_+9<data_.size()` instead of `offset_<data_.size()-9`. Reason: if `offset_==7` and  `data_.size()==8` then `data.size()-9` is a very big value due to wrap around of an unsigned, and `offset_<data_.size()-9` will be true. Then `data[offset_+1]` below will do an out of bound access on a string_view.

Reply:
Corrected.
The change was, at the original writing of the PR, conform the other tests. They were changed due to:
```
Commit: b58548e0bace2a8448891f22874e6c276804a0a5 [b58548e]

Date: Thursday, January 9, 2025 3:09:34 PM

issue #11333 rebuilding fedora ignition-transport causes abort in isExplicitPage with doxygen 1.13.1
```